e2e/cypress/integration/emoji/react_to_last_message_shortcut_spec.js

```diff
@@ -667,5 +667,5 @@ function openMainMenuOptions(menu) {
 function openChannelMainOptions(menu) {
     cy.get('body').type('{esc}');
     cy.findByLabelText('channel menu').click();
-    cy.findByText(menu).scrollIntoView().click();
+    cy.findByText(menu).scrollIntoView().should('be.visible').click();
```

Comment:
Not sure if this would fix the issue found on cypress server. It seems that on cypress server, it didn't scroll into view even though locally it works.

Reply:
Will try to run again but I've got the same failing test on local.